client/styles/abstracts/_placeholders.scss

```diff
@@ -174,13 +174,15 @@
 		color: getThemifyVariable('inactive-text-color');
 		cursor: pointer;
 		& g, & path {
-			fill: getThemifyVariable('inactive-text-color');
+			// fill: getThemifyVariable('inactive-text-color');
+			fill: #fff;
 		}
 		&:hover {
 			text-decoration: none;
 			color: getThemifyVariable('heavy-text-color');
 			& g, & path {
-				fill: getThemifyVariable('heavy-text-color');
+				// fill: getThemifyVariable('heavy-text-color');
+				fill: #fff;
```

Comment:
It seems like the reason that you changed this was because the logo was showing the wrong color on hover after you turned it into a link.  So we'll need to fix that.

This probably isn't the right place to change it as this `%link` mixin is used in multiple components so changing the color here may have unintended consequences (though I don't know off the top of my head which other links have svgs, maybe they aren't any 🤷).

Probably what we'll want is to add a `hover` style in the `_nav.scss` file which is specific enough to override this, just for the logo and not for other links.